### PR TITLE
add gofmt

### PR DESCRIPTION
**Pull request by svenwltr**
* adds a test that ensures the project is formatted properly
* adds a `format` command so the whole project can be formatted properly

I accidentally pushed to `master` :-( I reverted the commit and changed `master` to a protected branch.

### CODE REVIEW

**Comment by svenwltr** [comment]
@rebuy-de/it-platform Please review.

File: assets/build.sh [hunk 1]
```diff
@@ -37,10 +37,19 @@ function prepare {
...
+	then
+		echo "Source code isn't properly formated. Try 'build.sh format'."
+		exit 1
+	fi
 	/go/bin/gocov test -v $(glide nv) \
 		| /go/bin/gocov-html > ${OUTPUT_DIR}/coverage.html
 }
 
+function format {
+	gofmt -w $( glide nv -x )
```
Comment (bjoernhaeuser) [inline assets/build.sh]: my glide version doesnt know that command:

```
~ > glide nx
[ERROR]	Command glide-nx does not exist.
~ > glide --version
glide version 0.12.2
```

Comment (svenwltr) [inline assets/build.sh]: `glide nv -x` != `glide nx`

Comment (bjoernhaeuser) [inline assets/build.sh]: me blind!

**Pull request merged**